hadoop-ozone/dist/src/shell/conf/ozone-shell-log4j.properties

```diff
@@ -31,4 +31,4 @@ log4j.logger.org.apache.hadoop.util.NativeCodeLoader=ERROR
 log4j.logger.org.apache.ratis.conf.ConfUtils=WARN
 log4j.logger.org.apache.hadoop.security.ShellBasedUnixGroupsMapping=ERROR
 log4j.logger.org.apache.ratis.grpc.client.GrpcClientProtocolClient=WARN
-log4j.logger.org.apache.hadoop.io.retry.RetryInvocationHandler=WARN
+log4j.logger.org.apache.hadoop.io.retry.RetryInvocationHandler=WARN
```

Comment:
Nit: This file now only has whitespace change.  Can you please omit this from the PR?

Reply:
Understood will do !!